sanowret-crystal.lic

```diff
@@ -37,7 +38,8 @@ class SanowretCrystal
     return if DRStats.concentration < 100
     return if DRSkill.getxp('Arcana') >= 25
     return if hiding? || invisible?
-    return if Script.running?('sew') || Script.running?('carve') || Script.running?('forge') || Script.running?('remedy') || Script.running?('shape') || Script.running?('outdoorsmanship') || Script.running?('combat-trainer')
+    return unless @sanowret_during_combat && Script.running?('combat-trainer')
```

Comment:
❓ Are you sure this is the right logic? If I did my math right, this only runs the crystal logic if `sanowret_during_combat ` is true _and_ `combat-trainer` is running.

I expected something like `return if !@sanowret_during_combat && Script.running?('combat-trainer')` instead